translations/binaries/slps_strings_locations.json

```
@@ -0,0 +1,2636 @@
+        {
+            "offset": 1721416,
+            "string": "ダイクロフト(クリスタル)[END]",
+            "original_bytes_length": 25,
+            "available_bytes_size": 31,
+            "translation": "다이크로프트 (크리스탈)[END]"
+        },
+        {
+            "offset": 1721448,
+            "string": "ダイクロフト(第１階層)[END]",
+            "original_bytes_length": 23,
+            "available_bytes_size": 23,
+            "translation": "다이크로프트 (제 1 계층)[END]"
+        },
+        {
+            "offset": 1721472,
+            "string": "ガスクラウド戦[END]",
+            "original_bytes_length": 15,
+            "available_bytes_size": 15,
+            "translation": "가스 클라우드 전투[END]"
+        },
+        {
+            "offset": 1721488,
+            "string": "トラッシュマウンテン(後半)[END]",
+            "original_bytes_length": 27,
+            "available_bytes_size": 31,
+            "translation": "트래쉬 마운틴 (후반)[END]"
+        },
+        {
+            "offset": 1721520,
+            "string": "トラッシュマウンテン(前半)[END]",
+            "original_bytes_length": 27,
+            "available_bytes_size": 31,
+            "translation": "트래쉬 마운틴 (전반)[END]"
+        },
+        {
+            "offset": 1721552,
+            "string": "ディムロス戦[END]",
+            "original_bytes_length": 13,
+            "available_bytes_size": 15,
+            "translation": "딤로스 전투[END]"
+        },
+        {
+            "offset": 1721568,
+            "string": "ダリルシェイド・天上軍兵器戦２[END]",
+            "original_bytes_length": 31,
+            "available_bytes_size": 31,
+            "translation": "다릴 쉐이드・천상군 무기 전투 2[END]"
+        },
+        {
+            "offset": 1721600,
+            "string": "ダリルシェイド・天上軍兵器戦１[END]",
+            "original_bytes_length": 31,
+            "available_bytes_size": 31,
+            "translation": "다릴쉐이드・천상군 무기 전투 1[END]"
+        },
+        {
+            "offset": 1721632,
+            "string": "クレスタ・天上軍兵器戦[END]",
+            "original_bytes_length": 23,
+            "available_bytes_size": 23,
+            "translation": "클레스타・천상군 무기 전투[END]"
+        },
+        {
+            "offset": 1721656,
+            "string": "ソーディアン研究所[END]",
+            "original_bytes_length": 19,
+            "available_bytes_size": 23,
+            "translation": "소디언 연구소[END]"
+        },
+        {
+            "offset": 1721680,
+            "string": "アクアヴェイルF3[END]",
+            "original_bytes_length": 17,
+            "available_bytes_size": 23,
+            "translation": "아쿠아베일 F3[END]"
+        },
+        {
+            "offset": 1721704,
+            "string": "カルバレイスF3(ジャンクランド周辺)[END]",
+            "original_bytes_length": 35,
+            "available_bytes_size": 39,
+            "translation": "칼바레이스 F3 (정크랜드 주변)[END]"
+        },
+        {
+            "offset": 1721744,
+            "string": "カルバレイスF3(カルビオラ周辺)[END]",
+            "original_bytes_length": 31,
+            "available_bytes_size": 31,
+            "translation": "칼바레이스 F3 (칼비올라 주변)[END]"
```

```suggestion
            "translation": "칼바레이스 F3 (까르비올라 주변)[END]"
```

이거 혹시 까르바레이스.....가 되야하는 것은 아니겠죠?

---

이쪽은 확실하게 칼바레이스가 맞습니다
파일이 더 있는 줄 알았으면 이쪽까지 해두는건데 못 봤네요 고생하셨습니다...!